feature-libs/my-account/organization/components/order-approval/details/order-approval-detail-form/order-approval-detail-form.component.spec.ts

```diff
@@ -219,6 +212,26 @@ describe('OrderApprovalDetailFormComponent', () => {
         comment: testComment,
       }
     );
+
+    getOrderApproval$.next({
+      ...mockOrderApproval,
+      approvalDecisionRequired: false,
+    });
+    fixture.detectChanges();
+    assertBackButtonExist();
+    assertButtonAbsent('orderApproval.showForm_APPROVE');
+    assertButtonAbsent('orderApproval.showForm_REJECT');
+  }
+
+  function assertBackButtonExist() {
+    const backToListLink: DebugElement = el
+      .queryAll(By.css('a'))
+      .find((button) =>
+        (button.nativeElement as HTMLElement).textContent.includes(
+          'orderApproval.back'
+        )
+      );
+    expect(backToListLink).toBeTruthy('back to list button should be visible');
```

Comment:
cool, never knew we can add like a description to it until I saw this and made me look at the function definition of toBeTruthy 🔥